packages/wordpress-theme-boilerplate/lib/commands/create.js

```diff
@@ -0,0 +1,280 @@
+'use strict';
+
+const { promisify } = require( 'util' );
+const fs = require( 'fs' ).promises;
+const simpleGit = require( 'simple-git/promise' );
+const { Octokit } = require( '@octokit/rest' );
+const inquirer = require( 'inquirer' );
+const keytar = require( 'keytar' );
+const replace = require( 'replace-in-file' );
+const { pascalCase, camelCase, paramCase, snakeCase } = require( 'change-case' );
+const terminalLink = require( 'terminal-link' );
+const rimraf = promisify( require( 'rimraf' ) );
+const { log, format } = require( '../logger' );
+const { validateSlug, validatePHPNamespace, validateNotEmpty } = require( '../validation' );
+const { sleep, runShellCommand, runStep } = require( '../utils' );
+
+const TEMPLATE_OWNER = 'wearerequired';
+const TEMPLATE_REPO = 'wordpress-theme-boilerplate'; // @TODO
+const WORKING_DIR = process.cwd();
+
+async function create( command ) {
+	if ( ! command.skipIntro ) {
+		log(
+			format.title( '👋  Welcome to the WordPress Theme Boilerplate CLI Tool\n\n' ) +
+				'This tool will guide you through the setup process.\n' +
+				'Before you can start please make sure you have created a ' +
+				terminalLink(
+					'personal access token for GitHub',
+					'https://github.com/settings/tokens'
+				) +
+				" with the 'repo' scope selected.\nAfter the first run the token gets stored in your system's keychain and will be pre-filled on next runs.\n"
+		);
+
+		const { isReady } = await inquirer.prompt( [
+			{
+				type: 'expand',
+				name: 'isReady',
+				default: 2, // default to help in order to avoid clicking straight through
+				choices: [
+					{ key: 'y', name: 'Yes', value: true },
+					{ key: 'n', name: 'No', value: false },
+				],
+				message: 'Are you ready to proceed?',
+			},
+		] );
+
+		if ( ! isReady ) {
+			log( format.error( '\nAborted.' ) );
+			process.exit();
+		}
+
+		log();
+	}
+
+	// Get token from the keychain.
+	const storedGithubToken = await keytar.getPassword( 'wordpress-plugin-boilerplate', 'github' );
+
+	const {
+		githubToken,
+		projectName,
+		projectSlug,
+		projectURL,
+		themeName,
+		themeDescription,
+		themeSlug,
+		phpNamespace,
+		githubSlug,
+		privateRepo,
+	} = await inquirer.prompt( [
+		{
+			type: 'password',
+			mask: '*',
+			name: 'githubToken',
+			default: storedGithubToken,
+			message: 'GitHub API token:',
+			validate: validateNotEmpty,
+		},
+		{
+			type: 'input',
+			name: 'projectName',
+			default: 'Project Name',
+			message: 'Enter the name of the project:',
+			validate: validateNotEmpty,
+		},
+		{
+			type: 'input',
+			name: 'projectSlug',
+			default: ( answers ) => paramCase( answers.projectName ),
+			message: 'Enter the slug of the project:',
+			validate: validateNotEmpty,
+		},
```

Comment:
I think we should remove these too and use proper inputs for theme name and description. I don't see a problem in editing the PHP namespace to include a project specific one if it's necessary.

Reply:
The project slug is used here too: https://github.com/wearerequired/wordpress-theme-boilerplate/search?q=project-slug

So the namespace would be `Required\ThemeName\Theme`?

Reply:
No, the placeholder would just be `Required\ThemeName`.

But we could make the default value to something like  `'Required\\' + pascalCase( answers.themeSlug ) + '\\Theme'`, probably a bit smarter by removing a `-theme` suffix from the slug first.

The plugin boilerplate doesn't use a project slug for Traduttore either. We could a specific input for this which asks for the full path and defaults to theme/plugin slug.

Reply:
I don't fully understand why the placeholder should be `Required\ThemeName` and not `Required\ThemeName\Theme`.

Reply:
Because it allows you to customize the namespace for your own needs. As mentioned above the default value for our setup script will still include the `\Theme` part. But it would still allow me to use something like `Foobar\MyNiceTheme` or even `Foobar\Nniiccee` as a namespace.